packages/flutter/lib/src/material/app.dart

```diff
@@ -455,6 +457,94 @@ class MaterialApp extends StatefulWidget {
   /// {@macro flutter.widgets.widgetsApp.debugShowCheckedModeBanner}
   final bool debugShowCheckedModeBanner;
 
+  /// The default map of keyboard shortcuts to intents for the application.
```

Comment:
Should these docs maybe be a macro to be re-used between MaterialApp, CupertinoApp and WIdgetApp?

Reply:
They were, but when I added the example, I had to duplicate them.  You can't (yet) nest examples inside of macros in dartdocs, or it does weird things.

Reply:
Maybe I missed something,  but aren't the examples currently all the same?

The text before the examples could still be a macro, maybe?

Reply:
Yes, the examples are the same. I'll try and make the text before the examples and the see also into macros.